src/Twig/ExtensionV2/UserColorExtension.php

```diff
@@ -20,28 +20,37 @@ public function getFunctions(): array
         return [
             new TwigFunction('getUserBackgroundColor', [$this, 'getUserBackgroundColor']),
             new TwigFunction('getUserButtonColor', [$this, 'getUserButtonColor']),
+            new TwigFunction('getUserTextColor', [$this, 'getUserTextColor']),
+            new TwigFunction('getUserThemeColor', [$this, 'getUserThemeColor']),
         ];
     }
 
-    public function getUserBackgroundColor(): string
+    private function getUserColorClass(string $beneficiaryColor, string $proColor): string
```

Comment:
I think this function is a bit strange, the fact that it uses 2 arguments and chooses between 2 arguments is not specific to this function, so it seems wrongly named, maybee a more precise implementation would be
```php
private function getUserColorClass(string $prefix = ''): string {
        if (!$user = $this->getUser()) {
            return '';
        }

       return sprintf('%s%s', $prefix, $user->isBeneficiaire() ? 'green' : 'blue');
```

so that later, you can do 
```php
return $this->getUserColorClass('bg-light-');
return $this->getUserColorClass('bg-');
return $this->getUserColorClass('text-'');
return $this->getUserColorClass();
```

Reply:
You can also create a sub function
```php
$this->getUserColorName(User $user) {
    retur  $user->isBeneficiaire() ? 'green' : 'blue'
}
```

Reply:
Or also create a function inside user 
```php
$user->getColorName();
```